base/src/store.rs

```diff
@@ -0,0 +1,65 @@
+use crate::error::BlockError;
+use core::future::Future;
+use core::pin::Pin;
+use cid::Cid;
+use std::path::Path;
+
+pub type StoreResult<T> = Pin<Box<dyn Future<Output = Result<T, BlockError>> + Send>>;
+
+/// Visibility of a block.
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+pub enum Visibility {
+    /// Block is not announced on the network.
+    Private,
+    /// Block is only announced on the local network.
+    Local,
```

Comment:
Not sure how useful this option is. It doesn't provide much additional security. Data you want to share within a company network for example should be `Private` and the cid's communicated out-of-band.

Reply:
Removed this option to prevent accidental missuse. I think this feature is likely to be missused. Also I think it's an anti-pattern, data is public or it isn't. If the application wants to do this for performance reasons they should configure the store without a dht for example.